Wikipedia/Code/WMFImageTag.m

```diff
@@ -84,9 +88,14 @@ - (instancetype)initWithImageTagContents:(NSString*)imageTagContents {
         *stop = width && height && dataFileWidth && dataFileHeight && src && srcAttributeRange.location != NSNotFound;
     }];
 
+    //Don't continue initialization if we have invalid src
+    if([src length] == 0){
```

Comment:
We don't have to because its in the default initializer (thats why I added it there so we only need to do it once)